### ISSUE TITLE
Misleading log when using custom witnet.toml: "wallet is not connected to a node"

### ISSUE THREAD

**Issue by tmpolaczyk** [witnet/sheikah#1124]
When starting sheikah with a custom witnet.toml and a wallet already running, everything works fine but the following logs appear in the terminal:

```
$ ./dist_electron/sheikah-witnet-wallet-0.1.0.AppImage
Latest release version: 0.8.0
Running wallet...
stdout: HEADS UP! No configuration specified/found. Using default one!
Setting log level to: INFO, source: Config

Failed to register app protocol [Error: The scheme has been registered]
stderr: [2020-05-19T14:12:38Z ERROR witnet_wallet] No node url in config! To connect to a Witnet node, you must manually add the address to the configuration file as follows:
    [wallet]
    node_url = "127.0.0.1:21338"
    
Error: wallet is not connected to a node
```

Wallet section of `~/.sheikah/witnet.toml`:

```
[wallet]
testnet = true
node_url = "127.0.0.1:1234"
```

So the log is wrong but everything works fine, probably sheikah starts another wallet or something.

**Comment by Tommytrg** [comment]
I've tried to reproduce this but I couldn't.
![image](https://user-images.githubusercontent.com/24905681/83662587-fa9eaf80-a5c7-11ea-90a2-bc608cf10ed5.png)

Do you already have a wallet running? Maybe could be related with this witnet/witnet-rust/issues/1251.

Could you give more steps to try to reproduce it?

**Comment by tmpolaczyk** [comment]
> Do you already have a wallet running?

Yes, that's the point, I want to use sheikah with a wallet that's already running

**Comment by tmpolaczyk** [comment]
I find strange this line:

```
stdout: HEADS UP! No configuration specified/found. Using default one!
```

Because `~/.sheikah/witnet.toml` is automatically created by sheikah, how it is possible that it's not found?

**Comment by tmpolaczyk** [comment]
Can you write a step-by-step tutorial on how to run sheikah with a custom witnet.toml? Maybe I'm just doing it wrong

**Comment by Tommytrg** [comment]
Sheikah's bundle was thought to be used with the latest release of the wallet. In future maybe you could import a wallet path or something to achieve that. For the time being, you should be able to do it with:
- Run the bundle.
- Kill the wallet process started by sheikah.
- Run your own wallet.

On the other hand, I've fixed the log error you got `stdout: HEADS UP! No configuration specified/found. Using default one!` in #1178

**Comment by tmpolaczyk** [comment]
Perfect.

**Comment by Tommytrg** [comment]
Closed by #1178
